roles/osasinfra/templates/clientpod.yml.j2

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ cifmw_osasinfra_client_pod_name }}
+  namespace: openstack
+spec:
```

Comment:
We need to include inside the specs below:

`automountServiceAccountToken: false
`
https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting

Reply:
Added